### DISCUSSION TITLE
Columns Layout: Keep Columns Hidden when opening a dataframe?

### DISCUSSION THREAD

**Discussion by jacobmgreer** [posit-dev/positron#3987] [Q&A]
Is there a way to make the columns hidden by default when opening a dataframe? I see how to hide them when opening the data explorer, but unsure how to set them hidden as a default.

Also, I love the new data explorer in Positron! Amazing to filter on the fly!

**Comment by juliasilge** [comment]
I think this is a great idea! I have added it to this issue about that panel: https://github.com/posit-dev/positron/issues/2823

**Comment by luisDVA** [comment]
I support this idea. I was looking for a way to make "Summary Hidden" my default in the settings.json file but ended up here

edit: thanks for all the great work!

**Comment by ArtofmyApple** [comment]
I agree as well! I was trying to find a way of hiding the data summary and ended up here. If there was a way of implementing this idea under the "Layout" drop down options, then it would be great!

**Comment by Eiriksen** [comment]
+1 from me. It seems like the summary panel creates a bit of lag when opening; It would be nice to have a way to have it disabled by default, and then open when needed.

**Comment by juliasilge** [comment]
We are tracking this feature (provide a setting to hide the summary panel by default) in #5435. Please feel free to 👍 on that issue!
